masterlist.yaml

```diff
@@ -2993,6 +3172,33 @@ plugins:
     clean:
       - crc: 0xDD84DD4C # v2.1
         util: SSEEdit v3.2.1
+  - name: 'Cloaks.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/6369/'
+        name: 'Cloaks of Skyrim on Nexus Mods'
+    tag:
+      - name: Delev
+        condition: 'active("Unofficial Skyrim Special Edition Patch.esp")'
```

Comment:
Not sure if this is good practice, but it was an unusual issue where a levelled list entry removed by USSEP was being added back in by Cloaks.
Even though Cloaks has no Delev or Relev tags.
The expected bash output is only achieved if this Delev tag is present.

Reply:
So when the Bashed Patch is built with USSEP tagged as Delev and Cloaks without Delev, the Bashed Patch wasn't including the USSEP change? That's unexpected...

Reply:
No my mistake it was a Skyrim Immersive creature which added it back in, that has Delev & Relev so  it's expected behaviour.

![image](https://user-images.githubusercontent.com/1944639/42734701-6170d00a-8840-11e8-9605-f4a2be0e3d06.png)